### PR TITLE
Update to msal@1.2.1

### PR DESCRIPTION
**Pull request by sameerag**
Updating the CDN in the scripts to the latest released `msal@1.2.1`, release notes can be found [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/releases/tag/v1.2.1).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,8 +24,11 @@
   <!-- bluebird only needed if this page needs to run on Internet Explorer -->
   <!-- msal.min.js can be used in the place of msal.js; included msal.js to make debug easy -->
   <script src="https://cdnjs.cloudflare.com/ajax/libs/bluebird/3.3.4/bluebird.min.js" class="pre"></script>
-  <script src="https://secure.aadcdn.microsoftonline-p.com/lib/1.1.2/js/msal.js"></script>
-  <script src="https://code.jquery.com/jquery-3.2.1.min.js" class="pre"></script>
+  <script type="text/javascript" src="https://alcdn.msauth.net/lib/1.2.1/js/msal.js" integrity="sha384-9TV1245fz+BaI+VvCjMYL0YDMElLBwNS84v3mY57pXNOt6xcUYch2QLImaTahcOP" crossorigin="anonymous"></script>
```
Comment (jennyf19) [inline index.html:27]: >integrity= [](start = 85, length = 10)

what does this do?

**Review by jennyf19 — APPROVED** [review]
:shipit:

**Comment by jennyf19** [comment]
@sameerag  looks good to me, i ran it and called the web api. local account and FB. not sure if the issues attached are addressed though, with my very limited js understanding. lol

File: index.html (hunk 1 above)
Comment (sameerag) [reply index.html:27]: These are SRI hashes added for our CDN builds, more notes [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-core/README.md)

Comment (jennyf19) [reply index.html:27]: cool. thanks.

**Pull request merged**